app/src/components/ProfileComponent/ProfileComponent.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { View, Text, Image } from "react-native";
+import GlobalStyles from "../GlobalStyles";
+import styles from "./styles";
+
+const ProfileComponent = ({isPro}) => {
+  return (
+    <View style={[GlobalStyles.PrimaryBG, styles.profileContainer]}>
+      
+        <Image
+          style={styles.profilePicture}
+          source={require("../../../assets/alex.jpg")}
+        />
+
+        <Text style={isPro ? styles.isPro : {display: 'none'}}>S|Pro</Text>
+        <Text style={styles.name}>Alex Lazcano</Text>
+        <Text style={styles.email}>alex.lazcano@gmail.com</Text>
+
+        <Text style={styles.bio}>
+          Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet
+          sint. Velit officia consequat duis enim velit mollit. Exercitation
+          veniam consequat sunt nostrud amet.
+        </Text>
+      
+    </View>
+  );
+};
+
+export default ProfileComponent;
+
+{
+  /* <Text style={styles.isPro}>S|Pro</Text>
+<Text style={styles.name}>Alex Lazcano</Text>
+<Text style={styles.email}>alex.lazcano@gmail.com</Text>
+
+<Text style={styles.bio}>Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia consequat duis enim velit mollit. Exercitation veniam consequat sunt nostrud amet.</Text> */
+}
```

Comment:
Can this be removed? Or is this template code for something? Just thinking we might want to keep the code base clean if this isn't being documented for a special reason :smile:

Reply:
These changes were from a month ago so I don't remember why it commented out 😅. But I still need to change this component to a styled component so I will clean it up when I do